### DISCUSSION TITLE
Generating Readme stats

### DISCUSSION THREAD

**Discussion by ayushi77712** [anuraghazra/github-readme-stats#2584] [Q&A] [answered]
Can Somebody help me in generating readme stats and other stuff for my profile ?

**Comment by DURGESH716** [comment, marked as answer]
![Ayushi's GitHub stats](https://github-readme-stats-two-nu-79.vercel.app/api?username=ayushi77712&show_icons=true&theme=merko)

```md
![Ayushi's GitHub stats](https://github-readme-stats-two-nu-79.vercel.app/api?username=ayushi77712&show_icons=true&theme=merko)
```

<p><img align="center" src="https://github-readme-streak-stats.herokuapp.com/?user=ayushi77712&theme=merko" alt="ayushi77712" /> </p> 

```md
<p><img align="center" src="https://github-readme-streak-stats.herokuapp.com/?user=ayushi77712&theme=merko" alt="ayushi77712" /> </p> 
```
